includes/class-wc-subscriptions-order.php

```diff
@@ -2170,9 +2170,9 @@ public static function get_outstanding_balance( $order, $product_id ) {
 
 		$failed_payment_count = self::get_failed_payment_count( $order, $product_id );
 
-		$oustanding_balance = $failed_payment_count * self::get_recurring_total( $order, $product_id );
+		$outstanding_balance = $failed_payment_count * self::get_recurring_total( $order, $product_id );
 
-		return $oustanding_balance;
+		return $outstanding_balance;
```

Comment:
👍 Noting that I double checked these are the only 2 uses of this variable.

This is also a deprecated function.